sdk/tables/data-tables/test/internal/node/generateSas.spec.ts

```diff
@@ -37,6 +39,28 @@ describe("SAS generation", function () {
       assert.equal(tableSas, expectedSas1);
     });
 
+    it("should generate a SAS token with start partition and row keys", async () => {
```

Comment:
Mocha explicitly states that arrow functions are discouraged, because they cannot access the Mocha context available in `this`:

https://mochajs.org/#arrow-functions

Reply:
Well, I guess today is the day for Tables. I have migrated all the tests from arrow functions to anonymous functions.